src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -48,6 +50,8 @@ public PersonCard(Person person, int displayedIndex) {
         super(FXML);
         this.person = person;
         id.setText(displayedIndex + ". ");
+        System.out.println(person.getId().value);
```

Comment:
Again same thing happen here as discussed above:

Is print needed here? If not, consider to remove.